Documentation/MigrateToComposer/BestPractices.rst

```diff
@@ -46,7 +46,7 @@ Include Your Individual Extensions From `packages` Folder
 In the next step, you move all your individual extensions from
 :file:`public/typo3conf/ext` to :file:`packages`. And for this way to include them,
 it's important, that each extension has it's own correct :file:`composer.json` file.
-How this file should look in your extension, can be found on `composer.typo3.org <https://composer.typo3.org/>`__ or
+How this file should look in your extension, is documented  `Extension Development Documentation <https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ComposerJson/Index.html>`__ or
```

Comment:
You can cross-reference this instead:

    :ref:`t3coreapi:composer-json`

See https://docs.typo3.org/m/typo3/guide-installation/master/en-us/Linktargets.html and https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/Targets.html